### PR TITLE
adding /desks to back end

### PR DESCRIPTION
**Pull request by rebecalsantos**
updating recreate_schema.sql and populate_db.sql to match benjamin changes for desk positions.
Updating \api\desks to add the new colums

### CODE REVIEW

File: server/populate_db.sql [hunk 1]
```diff
@@ -5,12 +5,15 @@ INSERT INTO desk_user (username) VALUES ('Adeola');
...
-INSERT INTO desk (desk_name) VALUES ('desk 5');
-INSERT INTO desk (desk_name) VALUES ('desk 6');
+
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 1',0,0,0);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 2',175,0,180);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 3',0,170,0);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 4',175,170,180);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 5',0,340,0);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 6',175,340,180);
+];
```
Comment (Benjamin-GitHub) [inline server/populate_db.sql]: This data are correct. Thanks Rebeca!

File: server/api.js [hunk 2]
```diff
@@ -16,8 +16,25 @@ router.get("/", (req, res) => {
...
+	try {
+		const deskResult = await db.query("SELECT * from desk;");
+		// convert array with objects having {username, booking_date} into {name,desk_id,desk,date}
+
+		const desks = deskResult.rows.map((row) => {
+			return {
+				id: row.id,
+				desk: row.desk_name,
+				xCoor:row.x,
+				yCoor:row.y,
```
Comment (Benjamin-GitHub) [inline server/api.js]: Please check the rule of naming here "xCoor" & "yCoor". It seems you used different rule in line 50 => "desk_id " !?

**Review by Benjamin-GitHub — COMMENTED** [review]
Nice job Rebeca!

File: server/api.js (hunk 2 above)
Comment (rebecalsantos) [reply server/api.js]: this is already fixed in another pr. Many thanks for reviewing

**Pull request closed without merging**